### PR TITLE
Multiple website changes

### PR DESCRIPTION
**Pull request by Flip-HH**
- Add button in navbar for translating between french and english.
- Add tiers for new companies (hidden for now, simply unhide the components and add the new companies).
- Remove tint over Markhor pictures and videos
- Added Linkedin in contact page

### CODE REVIEW

File: src/plugins/i18n.ts [hunk 1]
```diff
@@ -195,4 +203,4 @@ const i18n = createI18n({
   allowComposition: true,
   messages,
 });
-export default i18n;
+export default i18n;
```
Comment (Leuchak) [inline src/plugins/i18n.ts:206]: revert this change pls

**Review by Leuchak — COMMENTED** [review]
good job, revert only the line feed change. Otherwise it will always change with each pull request.

**Review by Leuchak — APPROVED** [review]

**Comment by McCaroon** [comment]
Tested on mobile & on desktop. LGTM, are we ready to merge? @Flip-HH

**Comment by Flip-HH** [comment]
> Tested on mobile & on desktop. LGTM, are we ready to merge? @Flip-HH

Merging as we speak
